### DISCUSSION TITLE
Error when configuring firebase

### DISCUSSION THREAD

**Discussion by tomifis** [nuxt-community/firebase-module#579] [Q&A] [answered]
When trying to configure firebase this error appears (**Cannot read property 'apps' of undefined**)

**Comment by lupas** [comment, marked as answer]
@tomifis 
Which version of Firebase and the module are you using? Make sure you are not using Firebase v9.

**Reply by tomifis** [reply]
You're right, there is no support for Firebase v9 yet, thanks.
